src/kili/adapters/kili_api_gateway/label/annotation_to_json_response.py

```diff
@@ -79,10 +89,15 @@ def patch_label_json_response(self, label: Dict, label_id: LabelId) -> None:
             if not annotations and self._label_has_json_response_data(label):
                 return
 
-            annotations = cast(List[VideoAnnotation], annotations)
-            converted_json_resp = _video_label_annotations_to_json_response(
-                annotations=annotations, json_interface=self._project_json_interface
-            )
+            if self._project_input_type == "VIDEO":
+                annotations = cast(List[VideoAnnotation], annotations)
+                converted_json_resp = _video_annotations_to_json_response(
+                    annotations=annotations, json_interface=self._project_json_interface
+                )
+            else:
+                annotations = cast(List[ClassicAnnotation], annotations)
```

Comment:
`ClassicAnnotation` -> `StandardAnnotation`

Reply:
`ClassicAnnotation` would refer to the type names we have in the front-end & back-end as well, to keep consistency at the differences places of the code.